pages/common/az-appconfig.md

```diff
@@ -0,0 +1,29 @@
+# az appconfig
+
+> Manage App configurations on Azure.
+> Part of `az`, the command-line client for Microsoft Azure.
+> More information: <https://docs.microsoft.com/en-us/cli/azure/appconfig?view=azure-cli-latest>.
+
+- Create an App Configuration:
+
+`az appconfig create --resource-group {{rg_name}} --name {{appconfig_name}} --location {{location}}`
```

Comment:
We try to avoid abbreviations. I think if the `name` is first, the user knows is the appconfig name.
```suggestion
`az appconfig create --name {{name}} --resource-group {{group_name}} --location {{location}}`
```